doc/src/content/atoms/checkbox/SimpleCheckbox.md

```diff
@@ -0,0 +1,14 @@
+---
+path: "/checkbox/1"
+component: "checkbox"
+type: "atom"
+title: "Checkbox"
+---
+<codeblock>
+<Label htmlFor="terms">
```

Comment:
Is there a real effect of htmlFor in React ?

Reply:
Yes. we need it. It is helpful for associating a label with an input field. 
Especially we need it for Checkbox. because, if we want checkbox to state whenever label is clicked, we need `htmlFor`